generate-stackbrew-library.sh

```diff
@@ -17,14 +18,14 @@ echo '# maintainer: InfoSiftr <github@infosiftr.com> (@infosiftr)'
 for version in "${versions[@]}"; do
 	commit="$(git log -1 --format='format:%H' -- "$version")"
 	fullVersion="$(grep -m1 'ENV REDIS_VERSION ' "$version/Dockerfile" | cut -d' ' -f3)"
-	
+
 	versionAliases=()
 	while [ "$fullVersion" != "$version" -a "${fullVersion%[.-]*}" != "$fullVersion" ]; do
 		versionAliases+=( $fullVersion )
 		fullVersion="${fullVersion%[.-]*}"
 	done
 	versionAliases+=( $version ${aliases[$version]} )
-	
+
```

Comment:
Can we leave these tabs in?  We like empty lines in our blocks with the same indentation (unless it is in go, then `gofmt -w -s` is right).